swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+
+class Clothing (Item):
+    def __init__ (self, condition = 0, category = "Clothing"):
+        self.category = category
+        self.condition = condition
```

Comment:
Since you don't want the user to be able to override the category you should remove it from your parameters:

```suggestion
    def __init__ (self, condition = 0):
        self.category = "Clothing"
        self.condition = condition
```